database/migrations/2023_11_14_122252_add_type_column_to_collections_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Enums\TokenType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('collections', function (Blueprint $table) {
+            $table->string('type')->default(TokenType::Erc721->value);
+        });
+
+        DB::table('collections')->update([
+            'type' => TokenType::Erc721->value,
```

Comment:
some seeded collections are ERC-1155, won't this cause them to be incorrectly labeled?

Reply:
i'll look which of the seeded are erc1155 and hardcode this for them